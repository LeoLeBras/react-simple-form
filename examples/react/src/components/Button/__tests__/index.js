/* eslint padded-blocks: 0 */

import React from 'react'
import renderer from 'react-test-renderer'
import Button from './../'

describe('<Button /> component', () => {

  it('renders correctly', () => {
    const instance = renderer.create(
      <Button onClick={() => null}>
        Submit
      </Button>
    ).toJSON()
    expect(instance).toMatchSnapshot()
  })

})
