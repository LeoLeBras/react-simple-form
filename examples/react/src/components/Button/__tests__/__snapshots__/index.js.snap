exports[`<Button /> component renders correctly 1`] = `
<button
  onClick={[Function onClick]}>
  Submit
</button>
`;
